examples/pipelines/windows10-installer/README.md

```diff
@@ -0,0 +1,122 @@
+# Windows 10 Installer Pipeline
+
+Downloads a Windows Source ISO into a DV and installs Windows into a new base DV.
+It will then spin up an installation VM and use Windows Answer Files to automatically install the VM.
+Then the pipeline will wait for the installation to complete and will delete the installation VM while keeping the artifact DV with the installed operating system.
+The pipeline can be customized to support different installation requirements.
+
+There is a specific version of this pipeline for OKD. This version is using templates, which are not available on Kubernetes.
+
+## Prerequisites
+
+- KubeVirt `v0.53.1`
```

Comment:
exactly this version or at least this version?

Reply:
It only has to be a recent version of KubeVirt (can also be older). I chose this version as it was the latest version at the time of writing the docs.